<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorImage</class>
 <widget class="QWidget" name="InspectorImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>153</height>
   </rect>
  </property>
  <property name="accessibleName">
   <string>Image Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="panel" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="3">
       <widget class="QToolButton" name="resetAutoscale">
        <property name="toolTip">
         <string>Reset to default</string>
        </property>
        <property name="accessibleName">
         <string>Reset 'Scale to frame size' value</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="../musescore.qrc">
          <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="sizeHeight">
        <property name="accessibleName">
         <string>Height</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sizeWidth">
        <property name="accessibleName">
         <string>Width</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size:</string>
        </property>
        <property name="buddy">
         <cstring>sizeWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Scale:</string>
        </property>
        <property name="buddy">
         <cstring>scaleWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="scaleWidth">
        <property name="accessibleName">
         <string>Scale Width</string>
        </property>
        <property name="suffix">
         <string notr="true">%</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="scaleHeight">
        <property name="accessibleName">
         <string>Scale Height</string>
        </property>
        <property name="suffix">
         <string notr="true">%</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QToolButton" name="resetLockAspectRatio">
        <property name="toolTip">
         <string>Reset to default</string>
        </property>
        <property name="accessibleName">
         <string>Reset 'Lock aspect ratio' value</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="../musescore.qrc">
          <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QToolButton" name="resetSizeIsSpatium">
        <property name="toolTip">
         <string>Reset to default</string>
        </property>
        <property name="accessibleName">
         <string>Reset 'Size in staff space units' value</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="../musescore.qrc">
          <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="lockAspectRatio">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="accessibleName">
         <string>Lock aspect ratio</string>
        </property>
        <property name="text">
         <string>Lock aspect ratio</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="sizeIsSpatium">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="toolTip">
         <string>Distance between two lines on a standard 5-line staff</string>
        </property>
        <property name="whatsThis">
         <string>Distance between two lines on a standard 5-line staff</string>
        </property>
        <property name="accessibleName">
         <string>Size in staff space units</string>
        </property>
        <property name="text">
         <string>Size in staff space units</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="autoscale">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="accessibleName">
         <string>Scale to frame size</string>
        </property>
        <property name="text">
         <string>Scale to frame size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>autoscale</tabstop>
  <tabstop>resetAutoscale</tabstop>
  <tabstop>sizeWidth</tabstop>
  <tabstop>sizeHeight</tabstop>
  <tabstop>scaleWidth</tabstop>
  <tabstop>scaleHeight</tabstop>
  <tabstop>lockAspectRatio</tabstop>
  <tabstop>resetLockAspectRatio</tabstop>
  <tabstop>sizeIsSpatium</tabstop>
  <tabstop>resetSizeIsSpatium</tabstop>
 </tabstops>
 <resources>
  <include location="../musescore.qrc"/>
 </resources>
 <connections/>
</ui>
